objects/isn_sharedobjectscripts.lua

```diff
@@ -31,18 +31,18 @@ function isn_getAllDevicesConnectedOnNode(node,direction)
 end
 
 function isn_numericRange(number,minimum,maximum)
-	if number > maximum then return maximum end
-	if number < minimum then return minimum end
-	return number
+	if number > maximum then 
+    return maximum
+	elseif number < minimum then 
+    return minimum
+  else
+    return number
+  end
```

Comment:
This function is concise and readable and (probably) optimal as is. The only maybe-useful changes here would be to use built-in functions.